packages/react-native/scripts/cocoapods/utils.rb

```diff
@@ -196,7 +197,14 @@ def self.apply_xcode_15_patch(installer, xcodebuild_manager: Xcodebuild)
             end
             project.save()
         end
+    end
+
+    def self.apply_spm_dependencies(installer)
+        SPM.apply(installer)
+    end
```

Comment:
What about just use `SPM.apply(installer)` directly in `react_native_pods.rb`?

Given that we already encapsulated the SPM login in a class, this looks not necessary. It is just an extra indirection step and a maintenance cost.